backend/pages/templates/account/login.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load account socialaccount %}
+
+{% block head_title %}{% trans "Sign In" %}{% endblock %}
+
+{% block content %}
+
+<h1>{% trans "Sign In" %}</h1>
+
+<p>{% blocktrans %}If you have not created an account yet, then please
+<a href="{{ signup_url }}">sign up</a> first.{% endblocktrans %}</p>
```

Comment:
What's the purpose of allowing sign-ups?

Reply:
This is for the Facebook App review process so that their users can use the app to evaluate how it uses the Facebook information.